src/BuildScriptGeneratorCli/Commands/CommandBase.cs

```diff
@@ -43,6 +45,30 @@ public int OnExecute(CommandLineApplication app, IConsole console)
                 if (envSettings != null && envSettings.GitHubActions)
                 {
                     logger?.LogInformation("The current Oryx command is being run from within a GitHub Action.");
+
+                    // Format: "2020-02-15T02:51:50.000Z"
+                    var gitHubActionBuildContainerStartTime = envSettings.GitHubActionsBuildContainerStartTime;
+                    var gitHubActionBuildContainerEndTime = envSettings.GitHubActionsBuildContainerEndTime;
+
+                    var dateStart = gitHubActionBuildContainerStartTime.Split('T')[0];
```

Comment:
@qianz2 we should be able to avoid doing all of this parsing by using `DateTime.Parse(...)`

Reply:
Excellent suggestion!